packages/react-wrapper/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@empathyco/x-react-wrapper",
+  "version": "3.0.0",
+  "description": "A proxy that transforms Vue components into React ones",
+  "main": "dist/index.js",
+  "types": "types/index.js",
+  "files": [
+    "dist/**/*",
+    "types/**/*"
+  ],
+  "scripts": {
+    "serve": "parcel tests/demo/index.html",
+    "test": "npm run test:unit && npm run test:e2e:ci",
+    "test:unit": "jest",
+    "test:e2e": "start-server-and-test serve http://localhost:1234 cypress:open",
+    "test:e2e:ci": "start-server-and-test serve http://localhost:1234 cypress:run",
+    "cypress:open": "cypress open",
+    "cypress:run": "cypress run --headless --browser chrome",
+    "build": "tsc && npm pack",
+    "lint": "eslint --fix src",
+    "commit": "git-cz",
+    "release": "standard-version --no-verify",
+    "release:alpha": "standard-version --no-verify --prerelease alpha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://cbn-javieri@bitbucket.org/colbenson/react-wrapper.git"
+  },
+  "author": "",
```

Comment:
Sort these values as in the other files, for example in x-components:

![image](https://user-images.githubusercontent.com/6597815/126773253-e2cbe438-a6f5-4a76-a052-4bd4888def06.png)

Reply:
This task will be done in a future task. This was just to import the project (preserving the commits) and I've updated the `eslint` dependency so it does not install the older one (`empathy/eslint-plugin-x` instead of `empathyco/eslint-plugin-x`)